packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -553,14 +557,16 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
    * Allocate a new koid.
    *
    * @param {string} ownerID
-   * @param {undefined | bigint} id
```

Comment:
These changes look very familiar.  I think I almost did something like this once. :)

Reply:
Glad to see you changed this to `{bigint} [id]`. Resolving.